src/platform/site-wide/announcements/selectors.js

```diff
@@ -3,14 +3,48 @@ import moment from 'moment';
 // Relative imports.
 import _config from './config';
 
+// Checks if the announcement has started.
+const isStarted = announcement => {
+  const { startsAt } = announcement;
+
+  // Assume announcement is valid if startsAt was NOT provided.
+  if (!startsAt) {
+    return true;
+  }
+
+  // Derive if the announcement has started.
+  const startsAtDate = moment(startsAt);
+  const hasStarted = moment().isSameOrAfter(startsAtDate);
+
+  // Announcement has not started.
+  if (!hasStarted) {
+    return false;
+  }
+
+  // Announcement has started.
+  return true;
+};
+
 // Checks if the announcement has expired.
-const isExpiredAnnouncement = announcement => {
-  if (!announcement.expiresAt) return true;
+const isNotExpired = announcement => {
+  const { expiresAt } = announcement;
+
+  // Assume announcement is valid if expiresAt was NOT provided.
+  if (!expiresAt) {
+    return true;
+  }
 
-  const expiresAtDate = moment(announcement.expiresAt);
+  // Derive if the announcement has expired.
+  const expiresAtDate = moment(expiresAt);
   const hasExpired = moment().isSameOrAfter(expiresAtDate);
 
-  return !hasExpired;
+  // Announcement has not expired.
+  if (!hasExpired) {
+    return true;
+  }
+
+  // Announcement has expired.
+  return false;
```

Comment:
Optionally, I think you could reduce lines 41-47 by just returning `!hasExpired`. Could so something similar with the `isStarted` function.

Reply:
That's actually how it was originally but I felt that it was confusing to read `!hasExpired`. I can update it back to `!hasExpired` if you think it's more readable! 🙂

Reply:
I like the way the `hasStarted` ties into the `isNotExpired` function in the chain. I think it's good this way 👍